Sources/KeyboardShortcuts/Shortcut.swift

```diff
@@ -185,7 +185,30 @@ private var keyToCharacterMapping: [KeyboardShortcuts.Key: String] = [
 	.f17: "F17",
 	.f18: "F18",
 	.f19: "F19",
-	.f20: "F20"
+	.f20: "F20",
+	
+	// Representations for numeric keypad keys with   ⃣  Unicode U+20e3 'COMBINING ENCLOSING KEYCAP'
+	.keypad0: "0\u{20e3}",
+	.keypad1: "1\u{20e3}",
+	.keypad2: "2\u{20e3}",
+	.keypad3: "3\u{20e3}",
```

Comment:
A tiny problem with this is that the keycap style will not be shown in some places, like in a menu.

<img width="119" alt="Screenshot 2023-04-27 at 02 18 44" src="https://user-images.githubusercontent.com/170270/234680337-a9086307-5f88-47bd-96d6-ee9d0ee172d6.png">

Reply:
An alternative would be to use circled numbers: `"\u{2462}"`, but they are quite small...

<img width="343" alt="Screenshot 2023-04-27 at 02 19 56" src="https://user-images.githubusercontent.com/170270/234680604-8f8903b6-7d4a-41f4-ac0b-e4f2856bc1fd.png">


<img width="173" alt="Screenshot 2023-04-27 at 02 20 06" src="https://user-images.githubusercontent.com/170270/234680660-821f527b-192b-4698-aa52-7f5f8a60841d.png">

Reply:
I saw the circled numbers, found them to be too small.
The other alternative I see is plain strings like "keypad 3".

Reply:
If you have a keypad, try going to System Settings => Keyboard => Keyboard Shortcuts and see what Apple uses for these.